backend/Bindings/Edna.Bindings.LtiAdvantage/Services/NrpsClient.cs

```diff
@@ -111,5 +122,32 @@ public async Task<Member> GetById(string clientId, string tokenUrl, string audie
 
             return allMembers.FirstOrDefault(member => member.UserId.Equals(userId));
         }
+
+        private static string GetNextMembershipUrlFromHeaders(HttpResponseHeaders headers)
+        {
+            string linkUrls = headers.TryGetValues("Link", out var values) ? values.FirstOrDefault() : null;
+            if (string.IsNullOrEmpty(linkUrls))
+            {
+                return null;
+            }
+
+            string[] urls = linkUrls.Split(',');
+            string nextUrl = urls.FirstOrDefault(u => u.IndexOf("\"next\"", StringComparison.OrdinalIgnoreCase) >=0 );

```

Comment:
Possible to give an example of how the header looks like, I've taken a look at the spec but an actual example from something like a Moodle might be a better help in understanding.

Reply:
example header:
"<https://DOMAIN/api/lti/courses/1261/names_and_roles?page=1&per_page=50>; rel=\"current\",<https://DOMAIN/api/lti/courses/1261/names_and_roles?page=2&per_page=50>; rel=\"next\",<https://DOMAIN/api/lti/courses/1261/names_and_roles?page=1&per_page=50>; rel=\"first\",<https://DOMAIN/api/lti/courses/1261/names_and_roles?page=2&per_page=50>; rel=\"last\""

Reply:
Maitree's response: Wanted to perform case insensitive check, but string.contains does not support StringComparison.OrdinalIgnoreCase in the current .NET version. So used indexof for now.

Reply:
Ohh, makes sense. Thanks for explaining.